kclvm/sema/src/namer/node.rs

```diff
@@ -0,0 +1,341 @@
+use crate::core::package::{ImportInfo, ModuleInfo};
+use crate::core::symbol::{
+    AttributeSymbol, RuleSymbol, SchemaSymbol, SymbolKind, SymbolRef, TypeAliasSymbol,
+    UnresolvedSymbol,
+};
+
+use super::Namer;
+use kclvm_ast::ast;
+use kclvm_ast::walker::MutSelfTypedResultWalker;
+use kclvm_error::Position;
+
+fn get_node_position<T>(node: &ast::Node<T>) -> (Position, Position) {
```

Comment:
U can use the function related `Position` and `Range` defined in the `ast` module https://github.com/kcl-lang/kcl/blob/main/kclvm/ast/src/ast.rs#L65